static/js/lib/videojs_hls_quality_selector.js

```diff
@@ -0,0 +1,135 @@
+/* Adapted from https://github.com/chrisboustead/videojs-hls-quality-selector .
```

Comment:
I wasn't sure on whether this would be worth it. The original plugin just tests that it can register against videojs:
https://github.com/chrisboustead/videojs-hls-quality-selector/blob/master/test/plugin.test.js

We could write tests, but I think it would mostly look like a lot of mocking for videojs functions.

Do you think it's worth pursuing? Either way is fine with me.